pkg/types/validation/installconfig.go

```diff
@@ -36,6 +38,9 @@ const (
 	masterPoolName = "master"
 )
 
+// list of known plugins that require hostPrefix to be set
+var pluginsUsingHostPrefix = sets.NewString(string(operv1.NetworkTypeOpenShiftSDN), string(operv1.NetworkTypeOVNKubernetes))
```

Comment:
```suggestion
var pluginsUsingHostPrefix = sets.NewString("OpenShiftSDN", "OVNKubernetes")
```

Reply:
I think the installer uses these strings instead of operator types in other places so we should continue to do that for consistency

Reply:
Initially I had plain strings, until someone suggested I should use constants in the review for the CNO patch. I thought both my patches in CNO and installer should look the same, but if you strongly feel its important we should we use the plain strings I can flip it back. I understand that other places have strings like in line https://github.com/openshift/installer/blob/ce9161fbc004b6197c7c0bbfbc280101dceab416/pkg/types/validation/installconfig.go#L180 and https://github.com/openshift/installer/blob/ce9161fbc004b6197c7c0bbfbc280101dceab416/pkg/types/validation/installconfig.go#L207

but overall isn't it a better practice to use constants that ensure no typo ?

Reply:
If its any better, I could do a clean up patch to make sure we use constants wherever possible